src/scss/utils/theme-styles.scss

```diff
@@ -35,12 +35,26 @@ $default-theme-colors: (
   "white": #FFF,
 ) !default;
 
+//
+// Default light colors
+// --------------------------------------------------
+//
+// This is where all your light colours are declared.
+
+$default-light-colors: (
```

Comment:
Since this variable is only used for checking the name in the `theme-contrast-color` mixin we should use a shallow list rather than a map here. There's a great article outlining lists [here](https://www.sitepoint.com/sass-reference/lists/)

Reply:
done